### ISSUE TITLE
only removing the flashing purple screen in private mode

### ISSUE THREAD

**Issue by natask** [overdodactyl/ShadowFox#281]
I am not sure if this is the right place to ask but since ShadowFox solves this issue, I thought I can get some answers here.

under my current setup, I invert whole windows so white background that is so prevalent as default  across many applications and websites doesn't bother me.

It has its downsides; like having to re-invert images back but currently the one that has been bothering me is that Firefox private page has this purple flash with every opening of a new tab along a purple background for about:privatebrowsing. I was able to get about:privatebrowsing to my liking using usercontent.css but I can't figure out how to remove the flashing purple screen. 

I have tried out shadowFox and if I wasn't so in love with inverting windows directly, I would be using because along with all the dark theme it offers, it has solved this pesky issue. I am sure because the private page (and also new tab) defaults to a greyish brown color with no purple flashing (I have verified by continuously opening new private tabs).

### TL:DR
I was wondering which css line is responsible for the greyish brown background that removes the flashing purple screen in private mode so that I can adopt it for my usage.

Thank you.

**Comment by natask** [comment]
I couldn't wait longer so I just bit the bullet and did binary search to find the css lines responsible. For anyone curious they are located in userChrome.css:
```css
#tabbrowser-tabpanels {
  background: white !important
}
```
This affects both private and normal tabs. It is possible to affect them separately but I don't know how to nor is the knowledge a requirement for my use case. (maybe through @-moz-document ?)

**Comment by overdodactyl** [comment]
Glad you found a solution!
